File: exerciseManipArrays.py
# exercice 07. module 8
""" Sort it by alphabetical order.
    Display each song in it on its own line in the terminal.
    Add a 5th song at the end of the array.
    Display how many songs are in the array inside a sentence such as "There are x songs in the array."
    Remove the 3rd song in the array.
    Display the content of the array on one line.
    Display the length of the array.
"""

songs = ["Highway for Heaven", "Viens je t'emmène", "La corrida"]
print(songs[0])
print(songs[1])
print(songs[2])

songs.append(("The magistral wind"))
print(songs)

length_song = len(songs)
print("They are "+str(length_song)+" in the songs array")

songs.pop(2)
print(songs)

print(len(songs))

File: exerciseManipArrays.js
// exercice 10. module 8
/** 
    Create a JavaScript array to store your 3 favorite dish.
    List each of them in the console on their own separate line.
    Display the count of your favorite dishes.
    Add a 4th dish to the array.
    Display the count once more within a sentence such as "I have x favorite dishes."
    Remove the 2nd one!
    Sort them in alphabetical order.
    Display them once more but this time as an array in the console.
    Display the count again!
*/

var dishes = ["fondue", "raclette", "couscous"];
console.log(dishes);

//var fruits = ["oranges", "apples", "bananas"];
//console.log(fruits);

console.log(dishes[0]);
console.log(dishes[1]);
console.log(dishes[2]);

//var dishesLength = dishes.length
//console.log(dishesLength);
//console.log("I have "+dishesLength+" favorite dishes");

console.log(dishes.length);
console.log("I have "+dishes.length+" favorite amazing dishes")

dishes.splice(1,1);
console.log(dishes);

dishes.sort();
console.log(dishes);

console.log(dishes);

console.log(dishes.length);

